package.json

```diff
@@ -27,6 +28,7 @@
     "dev-app": "npm run build-lib && npm run dev --workspace=app",
     "dev-cms": "cd ./cms && npm run start",
     "dev-carbonmark": "concurrently --kill-others-on-fail \"npm:dev-lib\" \"npm run dev --workspace=carbonmark\"",
+    "dev-carbonmark-api": "npm run dev --workspace=carbonmark-api",
```

Comment:
Do we need a build script too?
For local development and for Vercel?

Reply:
My understanding is that the build process is handled by the [vercel github integration](https://vercel.com/klimadao/carbonmark-api/settings/git). If you check the scripts on the original repo there was no `build` script

https://github.com/KlimaDAO/carbonmark-api/blob/42e9df24f31563cacd0d5d9bc65fd34fc3f0d266/package.json#L9-L13

You're right though we might need to update this to be the same as the carbonmark deploy process since it's now in a monorepo.. what do you think?

Reply:
Ok looking in to this the process seems to be:

1. update the git integration (to point to our mono repo)
2. update the [root directory](https://vercel.com/docs/concepts/monorepos#using-monorepos-with-vercel-dashboard) setting
3. the serverless.js file will handle the rest

Reply:
hi @0xemc 
point 1,2,3 all correct.

but i wonder which settings will you update on vercel?
To link the new repo to the vercel deployment project, we should only run on the cli `vercel` and it ask us to link to the correct project.